src/components/Preferences/ScheduleSelector.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import ScheduleSelector from "react-schedule-selector";
+
+type PropsType = { color: string; semester: string };
+class Timetable extends React.Component<PropsType> {
+    state = { schedule: [], data: [], semester: this.props.semester };
+    today = new Date();
+    first = this.today.getDate() - this.today.getDay() + 1;
+    unselectedColorDict = {
+        Blue: "rgba(162, 198, 248, 1)",
+        Orange: "rgba(255, 165, 0)",
```

Comment:
maybe not a big deal right now but its hard to see the difference between the unselected and selected orange